config.xsd

```diff
@@ -89,6 +89,7 @@
         <xs:attribute name="reportInfo" type="xs:boolean" default="true" />
         <xs:attribute name="restrictReturnTypes" type="xs:boolean" default="false" />
         <xs:attribute name="limitMethodComplexity" type="xs:boolean" default="false" />
+        <xs:attribute name="triggerErrorBehaviour" type="TriggerErrorBehaviourType" default="default" />
```

Comment:
`behaviour` is a British spelling, while Psalm code usually uses American. However I would suggest to change it to `triggerErrorExits`, as `exits=always` seems clearer than `behavior=always` to me.

Reply:
Yeah, much as I wish everything were British I used American spellings (e.g. SwitchAnalyzer) very much on purpose since so many other programming things use American spellings (e.g. color)